pyiron_base/job/core.py

```diff
@@ -429,6 +430,16 @@ def remove_child(self):
         internal function to remove command that removes also child jobs.
         Do never use this command, since it will destroy the integrity of your project.
         """
+        # Check if the job requires to be removed from the full object (This is the case for external Storage)
+        try:
+            requires_full_object = self.get('REQ_OBJ_RM')
+        except ValueError as e:
+            requires_full_object = False
```

Comment:
I am not a fan of `try` and `except`, you could just check if the key exists in the HDF5 file

Reply:
True, will change this 👍

Reply:
```suggestion
        requires_full_object = self.get('REQ_OBJ_RM', default=False)
```
should work.

I'd like more descriptive name for the flag, though.